rest-service/manager_rest/security/user_handler.py

```diff
@@ -1,67 +1,159 @@
 from datetime import datetime
 import string
+import typing
 
-from flask import current_app
+from flask import current_app, Response, abort, Request
 from flask_security.utils import verify_password
 
+from cloudify.workflows import tasks
+from cloudify.models_states import ExecutionState
+
 from manager_rest.storage.models import Token, User
 from manager_rest.storage.models_base import db
 from manager_rest.manager_exceptions import (
     NoAuthProvided,
     UnauthorizedError,
 )
 from manager_rest.storage import user_datastore, get_storage_manager
-from manager_rest.execution_token import (set_current_execution,
-                                          get_current_execution_by_token,
-                                          get_execution_token_from_request)
-from manager_rest.utils import is_expired
+from manager_rest.execution_token import (
+    set_current_execution,
+    get_current_execution_by_token,
+    get_execution_token_from_request,
+    current_execution,
+)
+from manager_rest.security import audit
+from manager_rest.utils import (
+    is_expired,
+    is_sanity_mode,
+)
 
 
-ENCODED_ID_LENGTH = 5
```

Comment:
Nice catch- I noticed on friday that I forgot to remove that, was thinking of doing it at some point so I'm glad you nixed it.